tools/update-blog-dates.js

```diff
@@ -62,7 +62,9 @@ async function loadExistingDates() {
 	try {
 		return JSON.parse(await fs.readFile(DATES_FILE, "utf8"));
 	} catch (error) {
-		throw new Error(`Failed to parse ${DATES_FILE}: ${error.message}`);
+		throw new Error(`Failed to parse ${DATES_FILE}`, {
+			cause: error,
+		});
```

Comment:
I think we should either keep the original error message in the message, or change error logging. This way, JSON parse error wouldn't be printed.

https://github.com/eslint/eslint.org/blob/fe3d263903c37c3ba562ad4549e380445c67e4e2/tools/update-blog-dates.js#L123-L126

(the same comment for the other change in this file)